4_advanced_usage/77_distributed_rudder.txt

```diff
@@ -73,6 +73,106 @@ cd rudder-multiserver-setup
 
 ----
 
+==== Install rudder-db manually
+
+The setup is done in two places: on the database server and on the Rudder root server.
+It also allows moving an existing database to another server.
```

Comment:
Could you please add here an short clarification that this process is different from the rudder-multiserver approach and can be used just to use an external DB?